lib/stimulus_reflex/channel.rb

```diff
@@ -22,6 +22,7 @@ def receive(data)
     target = data["target"].to_s
     reflex_name, method_name = target.split("#")
     reflex_name = reflex_name.classify
+    reflex_name = reflex_name.end_with?("Reflex") ? reflex_name : compute_reflex_name(reflex_name)
```

Comment:
What about something like this?

```ruby
reflex_name = reflex_name.end_with?("Reflex") ? reflex_name : "#{reflex_name}Reflex"

# ...

reflex_class = reflex_name.constantize.tap { |klass| raise unless is_reflex?(klass) }
```

Reply:
This would work, but it has less functionality than what I proposed.

My version lets you define a class with any name that inherits from `StimulusReflex::Reflex`

So you could have a totally viable `Balloon << StimulusReflex::Reflex`

Reply:
Not sure I like that flexibility. Imagine a Rails controller not suffixed with “Controller”. I’d rather enforce convention on this.

Reply:
While I rarely find myself accessing controller consts directly, I agree with your point overall and then some. If the tables were turned, I probably wouldn't accept this logic. I do like putting sketches on the table in case something sticks. :)